bin/steps/appdynamics-setup

```diff
@@ -0,0 +1,76 @@
+BUILD_DIR='/app'
+echo "Setting up for Appdynamics"
+
+if ! grep -Fxq "appdynamics" /app/requirements.txt
+then
+  echo "appdynamics" >> /app/requirements.txt
```

Comment:
We still think the `requirements.txt` file should not be modified at all.

Our reference here is to the pattern we've previously followed for NewRelic's agent. Ruby and NodeJS are both ecosystems with a strong culture of explicit, declarative package management (through Bundler and npm). 

A dependency on AppDynamics, NewRelic or other agents is an *application* dependency, and it should be declared by the application itself.

pip has a well-understood, declarative `requirements.txt` file. We think users should declare the dependency there.